mllib/src/main/scala/org/apache/spark/ml/pmml/PMMLExportable.scala

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
(Uses the same public facing API as per the JIRA discussion re: lack of complaints from users with old API)